tests/e2e/hooks.ts

```diff
@@ -39,19 +38,6 @@ After(async function (): Promise<void> {
   await state.page.close()
 })
 
-const sendRequest = function ({ method, path }): Promise<any> {
-  const headers = {
-    Authorization: `Basic ${Buffer.from(`${config.adminUser}:${config.adminPassword}`).toString(
-      'base64'
-    )}`
-  }
-  return axios({
-    method,
-    url: join(config.baseUrlOcis, path),
-    headers
-  })
-}
-
 const deleteAllDicomFiles = async function (): Promise<void> {
```

Comment:
I think this function deletes only one file... therefore it would be more suitable to call it something like deleteDicomFile() or similar...